### ISSUE TITLE
Needs an option to stop prompting for permissions

### ISSUE THREAD

**Issue by AntiSol** [OMGSoundboard/android-app#86]
## :warning: Is your feature request related to a problem? Please describe
OMGSoundboard constantly requests permissions to modify system settings. I will never grant these permissions and never intend to use the features that require these permissions. The constant nagging to enable these permissions makes this app unusable for me, it needs a setting to disable this nagging.

## :bulb: Describe the solution you'd like
an options panel allowing me to tell it "stop nagging me for permissions"
Alternatively a "never" option on the nag screen so that it never asks me again.

## :bar_chart: Describe alternatives you've considered
Uninstall / use other Apps. This nag screen makes omgsoundboard unusable for me.

## :page_facing_up: Additional context
n/a

## :raising_hand: Do you want to develop this feature yourself?
- [ ] Yes
- [X] No

**Comment by OMGSoundboard** [comment]
Are you interested to work on this, @Rashxz?

**Comment by Rashxz** [comment]
Sure!

**Comment by AntiSol** [comment]
thanks!

**Comment by Rashxz** [comment]
Hey @AntiSol, this issue was addresed before (#51 ), are you using the latest version of the app?

**Comment by AntiSol** [comment]
version 11.0, the one in f-droid

**Comment by Rashxz** [comment]
Just test that one, the permission prompt it's only appearing once, as intended. Apart from that it only appears if you try to set a tune as ringtone/alarm/notification.

**Comment by AntiSol** [comment]
well that's weird. I just tried it again and it hasn't prompted me once.
It was still installed though, so I'm not trying from a fresh install. It definitely nagged me a bunch of times when I first installed it.
just a note that I'm using a de-googled phone, maybe that has something to do with it.
but since I can no longer replicate the problem feel free to close this if you want
thanks for your time looking into it.
